Database/Corrections/sodQuestFixes.lua

```diff
@@ -1883,6 +1883,10 @@ function SeasonOfDiscovery:LoadQuests()
         [82309] = { -- A Full Shipment
             [questKeys.specialFlags] = specialFlags.REPEATABLE,
         },
+        [82316] = { -- Seven Eyes I Seek...
```

Comment:
This quest should also be added to the rune quest list.

Reply:
https://github.com/Questie/Questie/pull/6147/commits/77146ea945b539ac399f9f89d2177c7739b98097